applications/dot_etl.md

```diff
@@ -0,0 +1,167 @@
+
+### Team Code Repos
+
+No team code repos at the moment.  Please see individual Github repos (below)
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/liangjh
+- https://www.linkedin.com/in/johncandido/
+
+
+## Development Status :open_book:
+
+We are currently in research phase;  Development / coding has not started on this project yet.
+
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:** TBD
+- **Full-Time Equivalent (FTE):**  0.25
+- **Total Costs:** 26,000 USD
+
+### Milestone 1 — ETL of Relay Chain + Google BigQuery Integration
+
+- **Estimated duration:** 3-4 months
+- **FTE:**  1
+- **Costs:** 8,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can set up and run the ETL to load data from Dot to offline / queryable formats |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article and detailed documentation that explains how to set up the basic Dot-ETL (audience: developers, etc).  We will also publish a few articles introducing the public dataset, a repository of queries and tutorials for a broader audience to get running with the public dataset immediately. |
+| 1. | Create SubQuery Managed Node | Utilize SubQuery framework to create a running indexer node on SubQuery's managed services, reading and indexing blocks on the Polkadot network (may involve a few iterations for testing) |
+| 2. | Set up BigQuery Schema to pull transaction data | Finalize stored format in BigQuery (double ledger format) that transactional data will be transformed into |
+| 3. | Airflow workflows to read SubQuery transaction updates | Read updates from SubQuery node via GraphQL queries and write to BigQuery on a periodic timeframe|
+| 4. | Deploy Airflow to Google Composer | Staging and Prod deployment pipelines for Airflow to GCP Composer |
+| 4. | Outreach to Google + Blockchain-ETL | After completion / testing, reach out to Google Cloud teams + Blockchain-ETL consortium to promote + add Dot to list of available / queryable / searchable chains.  There may be some infra items to complete here.  Will also coordinate w/ Web3 foundation. |
```

Comment:
I'm not sure that "outreach" can be covered by the grants program, as it is hard to evaluate. Can you explain how you would coordinate with us and what the infra items would be?

Reply:
Thanks, will update.  Infra likely means ongoing hosting of the bigquery dataset; believe we had discussed that we will not maintain the datasets, but release the foundational framework / code to do so.